src/MissionMode.cpp

```diff
@@ -249,26 +251,29 @@ void MandatoryBurns::transition_to()
 {
     sfr::rockblock::sleep_mode = true;
     sfr::acs::off = true;
-    sfr::imu::turn_off = true;
+    sfr::imu::turn_off = false;
 }
 
 void MandatoryBurns::dispatch()
 {
-    sfr::mission::current_mode = sfr::mission::regularBurns;
+    if(sfr::burnwire::mandatory_attempts > sfr::burnwire::mandatory_attempts_limit) {
+        sfr::mission::current_mode = sfr::mission::regularBurns;
+    }
+    
 }
 
 void RegularBurns::transition_to()
 {
     sfr::rockblock::sleep_mode = true;
     sfr::acs::off = true;
-    sfr::imu::turn_off = true;
+    sfr::imu::turn_off = false;
 }
 void RegularBurns::dispatch()
 {
     if (!sfr::button::pressed || !sfr::photoresistor::covered) {
```

Comment:
Should we make this an and? I don't know if we want to move to photo if the button might still be pressed or the photoresister might still be blocked. We could additionally check if these two readings are valid since that will cause them to be true all the time

Reply:
It's an OR in the wiki diagram, so maybe not